api/src/test/resources/io/kaoto/backend/api/resource/camel-conector-example.yaml

```diff
@@ -5,9 +5,9 @@ metadata:
 spec:
   source:
     uri: timer:earthquake
-    parameters:
```

Comment:
Parameters is not supported on the official Camel K classes. So I guess time to move on to properties.

Reply:
What is happening for previous routes using `parameters`?

Reply:
Moving to Camel K CRD that means we are only considering what is officially supported by that Camel K version. 

There is no previous Camel K CRD release so if we want to support legacy code. Do we want to keep doing the parsers ourselves?

For future Camel K CRD we may want to see what Camel K does. If it keeps backwards compatibility, that's good. If not... then we will have to decide what to do.